README.md

```diff
@@ -128,10 +133,28 @@ It is intended for **use during development**, and not in release builds or othe
 
 You can redact headers that contain sensitive information by calling `redactHeader(String)` on the `ChuckerInterceptor`.
 
+
 ```kotlin
 interceptor.redactHeader("Auth-Token", "User-Session");
 ```
 
+### Decode-Body 📖
+
+Chucker by default handles only plain text bodies. If you use a binary format like, for example, Protobuf or Thrift it won't be automatically handled by Chucker. You can, however, install a custom decoder that is capable to read data from different encodings.
+
+```kotlin
+object ProtoDecoder : BinaryDecoder {
+    fun decodeRequest(request: Request, body: ByteString): String? = if (request.isExpectedProtoRequest) {
+        decodeProtoBody(body)
+    } else null
```

Comment:
Nitpick: suggest to use curly braces in `else` part if we use them in `if` part.